src/lib/containers/MarkdownPopover.tsx

```diff
@@ -71,7 +72,9 @@ export const MarkdownPopover: React.FunctionComponent<MarkdownPopoverProps> = ({
         ref={setPopperElement}
         {...attributes.popper}
       >
-        <MarkdownSynapse {...contentProps} />
+        <Typography variant="body1">
+          <MarkdownSynapse {...contentProps} />
+        </Typography>
```

Comment:
We'll probably want to do this the other way around in the future but this works for now!